libconnman-qt/networkservice.cpp

```diff
@@ -138,6 +138,8 @@ class NetworkService::Private: public QObject
         CallGetProperties     = 0x00000020,
         CallGetProperty       = 0x00000040,
         CallSetProperty       = 0x00000080,
+        CallMoveAfter         = 0x00000100,
+        CallMoveBefore        = 0x00000200,
         CallAll               = 0x000000ff
```

Comment:
The call bits are not that much used (spotting only ::managed()), but in theory if there are new enums added, this CallAll should have enough bits turned on to include them?